src/robusta/runner/config_loader.py

```diff
@@ -151,7 +152,8 @@ def __load_playbooks_repos(
 
                 playbook_packages.append(playbook_package)
             except Exception as e:
-                logging.error(f"Failed to add playbooks reop {playbook_package} {e}")
+                logging.error(f"Failed to add playbooks repo {playbook_package} due to the following:")
```

Comment:
I think you can just use:

logging.error(f"Failed to ...", exc_info=True)

Reply:
True, it works. Changed.

Reply:
That's true, changed.